libtenzir/builtins/operators/to_write_save.cpp

```diff
@@ -361,6 +455,10 @@ class write_and_save_operator final
 private:
   std::unique_ptr<plugin_printer> printer_;
   std::unique_ptr<plugin_saver> saver_;
+
+  // This will anger @jachris if he sees it, but @eliaskosunen said it was okay.
+  // -- @dominiklohmann, half jokingly, during a Hackathon.
```

Comment:
Muha

Reply:
Resolving this quickly so he doesn't see.

Reply:
🫨

... 

Adding this to my to-be-deleted list 😉